byron/chain/executable-spec/src/Cardano/Spec/Chain/STS/Rule/SigCnt.hs

```diff
@@ -43,18 +37,13 @@ instance STS SIGCNT where
   transitionRules =
     [ do
         TRC ((pps, dms), sgs, vk) <- judgmentContext
-        let k = pps ^. stableAfter . to unBlockCount . to fromIntegral
-            t = pps ^. bkSgnCntT
+        let k  = pps ^. stableAfter . to unBlockCount . to fromIntegral
```

Comment:
Understandable it seems there is mismatch between the latest (?) version of this rule, and the implementation:

![image](https://user-images.githubusercontent.com/175315/57463124-7c63c280-727a-11e9-98eb-cf79696add29.png)

in particular `k` comes from the protocol parameters. Does this mean that we will model the global constants in this way? @mdimjasevic @nc6 any thoughts?

Reply:
I replied via chat (the Haskell implementation has been brought in line with the LaTeX version).